BGESexEstimation/BGESexEstimation.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+import csv
+
+def read_coverage_metrics(metrics_file_path):
+    covX = None
+    covY = None
+
+    with open(metrics_file_path, 'r') as file:
+        reader = csv.DictReader(file, fieldnames=['contig', 'num_reads', 'mean_coverage'])
+        for row in reader:
+            if row['contig'] == 'chrX':
+                covX = float(row['mean_coverage'])
+            elif row['contig'] == 'chrY':
+                covY = float(row['mean_coverage'])
+    return covX, covY
+
+def estimate_sex_ploidy(covX, covY):
+    lower_bound_xx = 0
```

Comment:
probably worth making these input parameters so that if we ever want to change them we don't need to change the source code of the script